windows-installer/stackdriver_logging_agent.nsi

```diff
@@ -0,0 +1,311 @@
+;--------------------------------
```

Comment:
Frankly, because we're automating (or plan on automating) builds, it's better to have the same entrypoint name, so that the scripts could be shared. I do understand the value of the more specific name, but automation trumps that, IMO, so let's change this to `setup.nsi`. @dhrupadb FYI.